sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    sagemaker_session = sagemaker.session.Session()
</pre><p> </p><p>Nit: the <code>sagemaker_session</code> seems to be used to get region, bucket etc., which can be replaces with  <code>pipeline_session</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
You're right, I don't actually need `sagemaker_session`.